targets/ath79-generic

```diff
@@ -76,3 +76,9 @@ device('ocedo-raccoon', 'ocedo_raccoon', {
 device('tp-link-archer-c6-v2', 'tplink_archer-c6-v2', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
+
+-- Ubiquity
+
+device('ubiquiti-nanostation-ac', 'ubnt_nanostation-ac', {
+	factory = false,
```

Comment:
Add the appropriate ath10k firmware package:

```
 	packages = ATH10K_PACKAGES_QCA9888,
```

Reply:
Done